src/WebJobs.Script.WebHost/Extensions/FunctionMetadataExtensions.cs

```diff
@@ -62,7 +62,7 @@ public static async Task<FunctionMetadataResponse> ToFunctionMetadataResponse(th
         /// <param name="functionMetadata">FunctionMetadata object to convert to a JObject.</param>
         /// <param name="config">ScriptHostConfiguration to read RootScriptPath from.</param>
         /// <returns>JObject that represent the trigger for scale controller to consume</returns>
-        public static async Task<JObject> ToFunctionTrigger(this FunctionMetadata functionMetadata, ScriptHostConfiguration config)
+        public static async Task<JObject> ToFunctionTrigger(this FunctionMetadata functionMetadata, ScriptHostOptions config)
```

Comment:
It's a PITA but it'd be good if we could fix the param names and help text for all these. Best to do this at the time of the renames since it's easier to catch/address then